src/Umbraco.Web/UmbracoHelper.cs

```diff
@@ -247,127 +193,87 @@ public IPublishedContent ContentSingleAtXPath(string xpath, params XPathVariable
 
-        private IEnumerable<IPublishedContent> ContentForObjects(IEnumerable<object> ids)
-        {
-            var idsA = ids.ToArray();
-            if (ConvertIdsObjectToInts(idsA, out var intIds))
-                return _publishedContentQuery.Content(intIds);
-            if (ConvertIdsObjectToGuids(idsA, out var guidIds))
-                return _publishedContentQuery.Content(guidIds);
-            return Enumerable.Empty<IPublishedContent>();
-        }
+        private IEnumerable<IPublishedContent> ContentForObjects(IEnumerable<object> ids) => _publishedContentQuery.Content(ids);
 
         /// <summary>
         /// Gets content items from the cache.
         /// </summary>
         /// <param name="ids">The unique identifiers of the content items.</param>
         /// <returns>The content items that were found in the cache.</returns>
-        public IEnumerable<IPublishedContent> Content(params int[] ids)
-        {
-            return _publishedContentQuery.Content(ids);
-        }
+        public IEnumerable<IPublishedContent> Content(params int[] ids) => _publishedContentQuery.Content(ids);
 
         /// <summary>
         /// Gets content items from the cache.
         /// </summary>
         /// <param name="ids">The keys of the content items.</param>
         /// <returns>The content items that were found in the cache.</returns>
-        public IEnumerable<IPublishedContent> Content(params Guid[] ids)
-        {
-            return _publishedContentQuery.Content(ids);
-        }
+        public IEnumerable<IPublishedContent> Content(params Guid[] ids) => _publishedContentQuery.Content(ids);
 
         /// <summary>
         /// Gets content items from the cache.
         /// </summary>
         /// <param name="ids">The unique identifiers, or the keys, of the content items.</param>
         /// <returns>The content items that were found in the cache.</returns>
         /// <remarks>Does not support mixing identifiers and keys.</remarks>
-        public IEnumerable<IPublishedContent> Content(params string[] ids)
-        {
-            return ContentForObjects(ids);
-        }
+        public IEnumerable<IPublishedContent> Content(params string[] ids) => _publishedContentQuery.Content(ids);
 
         /// <summary>
         /// Gets the contents corresponding to the identifiers.
         /// </summary>
         /// <param name="ids">The content identifiers.</param>
         /// <returns>The existing contents corresponding to the identifiers.</returns>
         /// <remarks>If an identifier does not match an existing content, it will be missing in the returned value.</remarks>
-        public IEnumerable<IPublishedContent> Content(IEnumerable<object> ids)
-        {
-            return ContentForObjects(ids);
-        }
+        public IEnumerable<IPublishedContent> Content(IEnumerable<object> ids) => _publishedContentQuery.Content(ids);
+
```

Comment:
Shouldn't this also be removed as it is now declared in **IPublishedContentQuery.cs**  and implemented in **PublishedContentQuery.cs**?

Reply:
Got the answer in https://github.com/umbraco/Umbraco-CMS/pull/7744#discussion_r386907044 -> resolving